mobify/bellows
# Scroll to header when single item open

Fixes #46 

Status: **Ready to merge**

Reviewers: @tedtate @kpeatt @jeffkamo 
## Changes
- Added scroll code when singleItemOpen is selected
- Fixed broken grunt task
## How to test-drive this PR
- checkout this branch
- `grunt serve`
- open the singleItemOpen example
  - add a huge ton of content to the second to last item
  - open the second to last item
  - open the last item
  - ensure the browser scrolls to the header of the last item
